docs/troubleshooting.md

```diff
@@ -12,4 +12,51 @@ Troubleshooting Guide
 
 ## Management
 
-TODO
+### Heketi refuses to start and the log lines contain the string:
+
+    "Heketi was terminated while performing one or more operations."
+
+In Heketi 6.0 support for pending operations were added to the system.
+These pending operations are logged in the db before Heketi
+configures gluster and then are removed when the action is
+complete. However, if the Heketi process is terminated while it is
+performing these actions the pending operations will remain
+in the db. Because these stale pending operations exist in the db
+Heketi (by default) refuses to start, in order to avoid many of
+these stale operation entries from piling up and giving the
+administrator an opportunity to clean up any half-completed
+items on the gluster side.
+
+In order to start Heketi again one can:
+* Set the environment variable `HEKETI_IGNORE_STALE_OPERATIONS=true`
+  (If provided the value should be "true" or "false").
```

Comment:
Nitpick: I think we should have a full stop before the opening parenthesis. And then a full stop before the closing one.
Also wouldn't it be "If provided, the value should..." (note the comma). I'm not too sure about English comma rules.